public/data/pranavarora1895.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Pranav Arora",
+  "type": "personal",
+  "bio": "Developer/Educator/Open Source Contributor",
+  "avatar": "https://github.com/pranavarora1895.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/pranavarora1895/",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PranavArora95",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/cosmicpranav007/",
+      "icon": "facebook"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
What do you think on capitalizing `i` here!?

```suggestion
      "name": "Follow me on LinkedIn",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant